vg/font_syscall_test.go

```diff
@@ -0,0 +1,20 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build !js
+
+package vg_test
+
+import (
+	"testing"
+
+	"github.com/gonum/plot/vg"
+)
+
+func TestVGFONTPATH(t *testing.T) {
+	if got := len(vg.FontDirs); got <= 0 {
```

Comment:
`got` cannot be negative; `len(vg.FontDirs) == 0`

and then `t.Fatal("zero length FontDirs")`